frontend/src/components/Admin/AddUser/AddUser.tsx

```diff
@@ -182,6 +254,27 @@ export default function AddUser(): JSX.Element {
                 </Button>
               </div>
             </Card.Content>
+            <Card.Content>
+              <div className={`ui one buttons ${styles.fullWidth}`}>
```

Comment:
UX improvement ideas: how about we only show the upload button if a file has been chosen already? And let's add some text here that says "Add/update users through CSV". And maybe let's even include a template file that the admin can download?